apps/bfd-model/bfd-model-rif-samples/src/main/java/gov/cms/bfd/model/rif/samples/StaticRifResourceGroup.java

```diff
@@ -130,13 +131,19 @@ private StaticRifResourceGroup(StaticRifResource... resources) {
   }
 
   /**
+   * Get Resources.
+   *
    * @return the related {@link StaticRifResource}s grouped into this {@link StaticRifResourceGroup}
    */
   public StaticRifResource[] getResources() {
     return resources;
   }
 
-  /** @return a {@link Set} of {@link RifFile}s based on {@link #getResources()} */
+  /**
+   * To Rif Files.
+   *
+   * @return a {@link Set} of {@link RifFile}s based on {@link #getResources()}
```

Comment:
In a case like this, the return is actually describing the description; could adjust it likeso:

```suggestion
   * Generates a {@link Set} of {@link RifFile}s based on {@link #resources}.
   *
   * @return the set of RIF files
```